<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="grayEasy">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enkel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mtTamW">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>MtTamW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ocean">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ocean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stillife">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stillife</string>
   </property>
  </widget>
  <widget class="QPushButton" name="grayAdvanced">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Avansert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="grayOriginal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Konvertering til gråtone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mtTamN">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>MtTamN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Konverterer et fargebilde til&lt;br/&gt;gråtonebilde.&lt;br/&gt;&lt;br/&gt;Velg hvilket bilde som skal&lt;br/&gt;konverteres til høyre.&lt;br/&gt;&lt;br/&gt;Benytt deg av enten den enkle&lt;br/&gt;versjonen for gråtonekonvertering&lt;br/&gt;ved å klikke på &amp;quot;enkel&amp;quot;, eller bruk &lt;br/&gt;den litt mer sofistikerte versjonen&lt;br/&gt;ved å klikke på &amp;quot;avansert&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="balls">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Balls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adjuster">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjuster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="grayCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trees">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Trees</string>
   </property>
  </widget>
  <widget class="QLabel" name="grayImg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>76</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../hdr-bilder/Adjuster/Adjuster_00032.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
